bundles/org.openhab.binding.visualcrossing/only_days.items

```diff
@@ -0,0 +1,623 @@
+Group Total_Weather_Data "Total Weather Data" [ "Equipment" ]
```

Comment:
What is the purpose of this file? It can be deleted. Same for `full_example.items`

Reply:
Check the Readme. I referenced those files there

Reply:
Ah, then they have to be moved to the doc subfolder. Let me check to be sure

Reply:
Yes, they need to be moed to the doc subfolder.

For reference:   https://www.openhab.org/docs/developer/guidelines.html#a-directory-and-file-layout

Reply:
done